### PR TITLE
Feat: Include utility-scale estimates in the examples

### PR DESCRIPTION
**Pull request by SebastianMorawiec**
## Description

Include description of feature this PR introduces or a bug that it fixes. Include the following information:

Added new example that includes utility scale problems

## Please verify that you have completed the following steps

- [x] I have self-reviewed my code.
- [x] I have included test cases validating introduced feature/fix.
- [ ] I have updated documentation.

### CODE REVIEW

**Review by AthenaCaesura — CHANGES_REQUESTED** [review]

File: examples/ex_11_utility_scale.py [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+        hardware_failure_tolerance=algorithm_implementation.error_budget.hardware_failure_tolerance,
+        decoder_model=decoder_model,
+    )
+    return gsc_resources, footprint_resources
+
+
+def save_to_file(gsc_resources, footprint_resources, lattice_type):
+    cwd = os.getcwd()
+    results_folder = get_new_directory(
+        lattice_type, save_base_path=cwd + "/new_results/"
```
[the remark this answers is not in the document]
Comment (AthenaCaesura) [reply examples/ex_11_utility_scale.py:97]: Perhaps it should be in the existing "data" folder.

File: examples/ex_11_utility_scale.py [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+        gsc_resources.extra.n_t_gates,
+        gsc_resources.extra.n_rotation_gates,
+        algorithm_implementation.error_budget.transpilation_failure_tolerance,
+    )
+
+    footprint_resources = get_physical_cost(
+        algorithm_implementation.program.num_data_qubits,
+        num_t=total_t_gates,
+        architecture_model=my_estimator.hw_model,
+        hardware_failure_tolerance=algorithm_implementation.error_budget.hardware_failure_tolerance,
```
[the remark this answers is not in the document]
Comment (SebastianMorawiec) [reply examples/ex_11_utility_scale.py:88]: I.... Don't know - I see it normally in both local version and on Github 😲 This is how style-fix formats it as well.

File: examples/ex_11_utility_scale.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (SebastianMorawiec) [reply examples/ex_11_utility_scale.py:97]: Now when Im thinking about it - we should allow users to set the path for the results IMHO. Ill make a new parameter for it

File: examples/ex_11_utility_scale.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (AthenaCaesura) [reply examples/ex_11_utility_scale.py:88]: It's strange that this isn't causing any errors.... But I guess it's probably ok. so long as the indentation works and looks ok.

**Review by AthenaCaesura — APPROVED** [review]

**Pull request merged**